code/api/migrations/0002_auto_20211129_0932.py

```diff
@@ -0,0 +1,146 @@
+# Generated by Django 3.2.8 on 2021-11-29 16:32
+
+from django.db import migrations
+import secrets
+from cmput404.constants import HOST
+
+
+def add_local_auth_credentials(apps, schema_editor):
+    Node = apps.get_model('api', 'Node')
+
+    # Add local credentials
+    local_secret = secrets.token_hex(10)
+    Node.objects.get_or_create(
+        host=HOST,
+        api_prefix="/api",
+        username="local",
+        password=local_secret,
+        remote_credentials=False
+    )
+    Node.objects.get_or_create(
+        host=HOST,
+        api_prefix="/api",
+        username="local",
+        password=local_secret,
+        remote_credentials=True
+    )
+
+
+def add_dummy_auth_credentials(apps, schema_editor):
+    Node = apps.get_model('api', 'Node')
+
+    # Don't run migration if on clone deployment
+    if HOST == "cmput404f21-team04.herokuapp.com":
+        return
+
+    # Add local dummy server (if running locally)
+    if "127.0.0.1" in HOST:
+        dummy_host = "127.0.0.1:8080" if ":8000" in HOST else "127.0.0.1:8000"
+
+        Node.objects.get_or_create(
+            host=dummy_host,
+            api_prefix="/api",
+            username="dummy",
+            password="secret",
+            remote_credentials=False
+        )
+        Node.objects.get_or_create(
+            host=dummy_host,
+            api_prefix="/api",
+            username="dummy",
+            password="secret",
+            remote_credentials=True
+        )
+
+
+def add_prod_auth_credentials(apps, schema_editor):
+    Node = apps.get_model('api', 'Node')
+
+    # Don't run migration if on clone deployment
+    if HOST == "cmput404f21-team04.herokuapp.com":
+        return
```

Comment:
Maybe we could add https://social-distribution-fall2021.herokuapp.com nodes here for the clone server?